process/smartContract/builtInFunctions/esdtFreezeWipe.go

```diff
@@ -66,7 +66,10 @@ func (e *esdtFreezeWipe) ProcessBuiltinFunction(
 	log.Trace(vmInput.Function, "sender", vmInput.CallerAddr, "receiver", vmInput.RecipientAddr, "token", esdtTokenKey)
 
 	if e.wipe {
-		acntDst.DataTrieTracker().SaveKeyValue(esdtTokenKey, nil)
+		err := acntDst.DataTrieTracker().SaveKeyValue(esdtTokenKey, nil)
```

Comment:
the error check here is useless because the function will only throw error if the value length is too big, but in your case the value will have the length 0 so no error will be returned. However, can remain like this for future updates in the `SaveKeyValue` function

Reply:
i know, but it is better to check the error here. As you do not know how the underlying implementation can change.